### ISSUE TITLE
FBReader download with wrong filename

### ISSUE THREAD

**Issue by Kuzorra22** [seblucas/cops#520]
Hello and a Happy Christmas !

I am using COPS in combination with FBReader. $config['cops_use_url_rewriting'] = "1" is set and when I download a book on my PC or Chrome on my Android mobile, it is downloaded with the correct filename. When I download the same book in FBReader it is downloaded as "fetch.php?....".

What am I doing wrong ?

YOS,
Kuzorra

**Comment by marioscube** [comment]
@Kuzorra22 

In the fbreader help I found:
`FBReader for Android and Windows/Mac/Linux supports a technology named OPDS that allows your reader to connect network e-book catalogues and stores. In the reader, you can browse and search over the network catalogues and download the found books directly.`

To download from COPS  (Calibre OPDS Php Server) using OPDS connect to COPS from FBreader with the url: http://<local_IP>/cops/feed.php

**Comment by Kuzorra22** [comment]
Yes, I know. But fetch.php is managing the download, not feed.php, and the download is ignoring the Content-Disposition in the header. Is that a FBReader issue ?

**Comment by marioscube** [comment]
@Kuzorra22

I installed FBReader (latest version) on MacOS.
I connected FBReader to COPS with http://192.168.1.xxx/cops/feed.php and everything works.
[ http://192.168.1.xxx/cops/ worked as well. ]

In COPS I have in config_local.php: $config['cops_use_url_rewriting'] = "0"

![Screenshot 2022-12-23 at 13 41 16](https://user-images.githubusercontent.com/20281601/209338338-17775b2e-f050-466e-befc-9017c38095d0.png)

Where does is go wrong for you?

**Comment by Kuzorra22** [comment]
But then I can only download when using the browser to access files and not view them. In the meantime I edited my config_local, so set the cops_use_url_rewriting to 0 when the client identifies as FBReader and 1 if else.

**Comment by marioscube** [comment]
@Kuzorra22 
So no more errors for you? :-)
